### PR TITLE
Changes supplier frontend to use generic templates

### PR DESCRIPTION
**Pull request by DilwoarH**

### CODE REVIEW

File: tests/app/main/test_notifications.py [hunk 1]
```diff
@@ -12,24 +12,6 @@ def teardown_method(self, method):
...
-        cookie_value = self.get_cookie_by_name(res, 'seen_user_research_message')
-        assert cookie_value is None
-
-    def test_should_not_see_banner_if_subscribed(self):
-        self.login(user_research_opted_in=True)
-        res = self.client.get('/suppliers')
-        assert res.status_code == 200
-        assert 'Help us improve the Digital Marketplace' not in res.get_data(as_text=True)
-        assert 'Sign up to be a potential user research participant' not in res.get_data(as_text=True)
-        assert 'class="user-research-banner-close-btn"' not in res.get_data(as_text=True)
```
Comment (katstevens) [inline tests/app/main/test_notifications.py]: Shouldn't the new base page contain these links?

Comment (lfdebrux) [reply tests/app/main/test_notifications.py]: It does, which is why I took these tests out, as I felt the behaviour should be tested in the frontend toolkit and functional tests, not here.

Comment (lfdebrux) [reply tests/app/main/test_notifications.py]: https://github.com/alphagov/digitalmarketplace-functional-tests/blob/master/features/user/user_research.feature

Comment (katstevens) [reply tests/app/main/test_notifications.py]: Makes sense 👍

**Pull request merged**